project.clj

```diff
@@ -53,5 +61,8 @@
   :lein-release {:scm :git
                  :deploy-via :lein-deploy}
 
-  :plugins [[lein-parent "0.3.1"]
-            [puppetlabs/i18n "0.8.0"]])
+  :plugins [[lein-parent "0.3.5"]
```

Comment:
One other "give a mouse a cookie" kinda request. I'm working with achin to release lein-parent 0.3.7 which will remove some warnings that are output when using it with newer lein versions. It'd be nice to wait until Monday and then bump this to 0.3.7 (see https://github.com/achin/lein-parent/pull/20) though that can totally be done in a separate PR if needed.

Reply:
Yeah, we can do that